spring-cloud-kubernetes-integration-tests/spring-cloud-kubernetes-client-loadbalancer-it/src/main/java/org/springframework/cloud/kubernetes/client/loadbalancer/it/KubernetesClientLoadBalancerApplicationIt.java

```diff
@@ -49,13 +54,15 @@ public static void main(String[] args) {
 
 	@Bean
 	@LoadBalanced
-	RestTemplate restTemplate() {
-		return new RestTemplateBuilder().build();
+	WebClient.Builder client() {
+		return WebClient.builder();
 	}
 
-	@GetMapping("/servicea")
+	@GetMapping("/loadbalancer-it/servicea")
```

Comment:
I can't seem to find a way for traefik to re-write the url, like nginx used to do before. But that is not the point of the test anyway, so no big deal, imo.